tests/pr_test.go

```diff
@@ -0,0 +1,114 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/cloudinfo"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+const basicExampleTerraformDir = "examples/kp-key"
+const resourceGroup = "geretain-test-observability-instances"
```

Comment:
This needs to be updated to a dedicated resource group for the module - eg: geretain-test-kms.

Reply:
Pointing to dedicated resource group called geretain-test-kms